packages/app-root/src/app/about/faq/page.js

```diff
@@ -0,0 +1,12 @@
+import { FAQ } from '@zooniverse/content'
+
+export const metadata = {
+  title: 'FAQ',
+  description: 'Frequently asked questions about The Zooniverse'
+}
```

Comment:
The `metadata` object generates a `<head>`, but can only be used in a server component 🙄 In each page.js, I'm importing a "screen" component from lib-content, which has its own `'use client'` directive.

This means the `<head>` content in app-root is not translatable yet.